sklearn/metrics/_classification.py

```diff
@@ -2461,3 +2461,157 @@ def brier_score_loss(y_true, y_prob, *, sample_weight=None, pos_label=None):
             pos_label = y_true.max()
     y_true = np.array(y_true == pos_label, int)
     return np.average((y_true - y_prob) ** 2, weights=sample_weight)
+
+
+def calibration_error(y_true, y_prob, sample_weight=None, norm='l2',
+                      n_bins=10, strategy='uniform', pos_label=None,
+                      reduce_bias=True):
+    """Compute calibration error of a binary classifier.
+
+    Across all items in a set of N predictions, the calibration error measures
+    the aggregated difference between (1) the average predicted probabilities
+    assigned to the positive class, and (2) the frequencies
+    of the positive class in the actual outcome.
+
+    The calibration error is only appropriate for binary categorical outcomes.
+    Which label is considered to be the positive label is controlled via the
+    parameter pos_label, which defaults to 1.
```

Comment:
The default `pos_label` is not 1 anymore.

Reply:
Good catch, thanks !